### DISCUSSION TITLE
FFB Delay on G920, fix possible on Assetto corsa, not so much in other titles

### DISCUSSION THREAD

**Discussion by Kireand** [berarma/oversteer#179] [Ideas]
So I was trying to get my G920 set up and I noticed that in Assetto corsa my FFB was *absurdly* delayed, but by using an option called "Skip FF steps" I was able to get it to a mostly normal feeling, but this option does not exist for other games like Project Cars 2 (Just an example), and all it really does is reduce the wheel's refresh rate in HZ (Anything bellow 133hz was more than fine), I was wondering if implementing this would be possible?

**Comment by berarma** [comment]
It's not possible to implement here. It's implemented as a workaround in my other project ffbtools and included in my improved driver for the legacy/PSX Logitech wheels.

**Reply by Kireand** [reply]
Damn that sucks, having everything in one app would've been awesome
But I'll try the FFBTools you mentioned

**Comment by Kireand** [comment]
Heyo just wanted to add a bit to this about the FFB delay, apparently on linux the steering wheel defaults to the Xbox mode firmware, after suddenly having an issue with it and it stopped detecting my wheel I installed a package called "usb_modeswitcher" and it fixed the update rate delay issue, now I can run at 333hz just fine without any delay, so it's now properly in PC mode and not Xbox (I can tell because now it's not listed as "Logitech G920 racing wheel USB" on games but rather just as "Logitech G920", which is the same as in windows)

**Reply by berarma** [reply]
I don't think it can work in XBox mode. It's the first time I've heard such thing. It might be worth checking with other users.

**Reply by Kireand** [reply]
Found some sources dating back to like 2016 on it after searching "G920 usb_modeswitch"
https://www.beamng.com/threads/running-on-linux-w-wheel-a-guide.74493/
https://www.draisberghof.de/usb_modeswitch/bb/viewtopic.php?t=2510

**Reply by berarma** [reply]
Yes, that's very well known, but I haven't ever heard the G920 could be used in XBox mode. There's many users (everyone?) using the G920 in PC mode and having FFB lag issues.
